include/odp/api/spec/timer_types.h

```diff
@@ -372,6 +378,16 @@ typedef struct {
 	/** Number of timers in the pool. */
 	uint32_t num_timers;
 
+	/** Pool priority
+	 *
+	 * Timers allocated from pools with a higher priority value are served with higher priority
+	 * than timers from lower priority pools. The prioritization algorithm is implementation
```

Comment:
Suggesting minor clarification (usage of term "value"):  ".. than timers from lower priority pools." ->  "... than timers allocated from pools with lower priority value."

Reply:
Changed in V3.